source/CardGames.py

```diff
@@ -334,3 +336,21 @@ def Play(dealer: Dealer, players: list):
         players.pop(players.index(player))
       input("Next Player press 'Enter' when ready!")
   return players_passed
+
+class Pot:
+  def __init__(self, money):
+    self.money = money
+
+  def addPot(self, amount):
+    self.money += amount
+
+  def addPot(self, amount, player):
+    if(player.money >= amount):
+      self.money += amount
+    else:
+      print("Error: %s attempted to bet more than they have" % player.name)
+    player.makeBet(amount)
```

Comment:
Here the "makeBet" already checks to make sure that the player has enough money and will print a message if they don't. This might be a redundancy that can removed if necessary.

Also here it seems like the makeBet function will be called no matter whether the if or else statements are executed. Wouldnt you want makeBet() to only be called in the 'if' instance?

Reply:
this looks good, I would just consider moving the player.makeBet line up into the "if" instance like Roy said.

Reply:
I don't see any issues with the code but maybe just include a period at the end for styling purposes.